docs/usage.rst

```diff
@@ -253,21 +253,26 @@ Regular HTML tags
 -----------------
 
 Sometimes some regular HTML tags are needed. For this the Html widget can be used.
+The attributes of the HTML tag can be accessed through the :code:`attributes` trait.
 
 Vuetify:
 
 .. code-block:: html
 
     <v-container>
-        <h1>My heading</h1>
+        <h1 title="I'm a title">My heading</h1>
     </v-container>
 
 ipyvuetify
 
 .. jupyter-execute::
 
     v.Container(children=[
-        v.Html(tag='h1', children=['My heading'])
+        v.Html(
+            tag='h1', 
+            attributes={'title': "I'm a title"},
```

Comment:
We try to be consistent in using single quotes for python strings.

Reply:
I'm sorry, I overlooked the single quote in the string. Makes sense to use double quotes then. (no need to change your solution to this)